src/account/accountActions.js

```diff
@@ -13,9 +13,10 @@ export const switchAccount = (index: number): Action => ({
   index,
 });
 
-export const realmAdd = (realm: string): Action => ({
+export const realmAdd = (realm: string, zulipVersion: string | null): Action => ({
```

Comment:
This seems like it should take the entire settings object, and let the reducers pick out what they want.

(Mostly because that'll be easier to extend later. But that may be another way of saying that "what parts of the server settings object will be stored" is an implementation detail that doesn't need to be exposed here.)

Reply:
Sure, that reasoning makes sense. Let's ping @gnprice, though, because I think he disagrees, reasoning that we only need the zulipVersion now, and that the barrier to extending it later is fairly low.